profile.html

```diff
@@ -56,16 +59,18 @@
         <div class="row my-4">
           <div class="col-lg-5">
             <div class="text-center">
-              <h1 class="p-3 display-6 border rounded border-2 shadow">
-                Daniel Pybus Sollid
+              <h1 class="p-3 display-6 border rounded border-2 shadow" id="profile-name">
+                Non existing person
               </h1>
+              <div id="user-photo">
               <img
                 src="img/Character.png"
                 alt="user photo"
                 class="mt-4"
                 style="width: 60%"
               />
             </div>
+            </div>
             <div class="border rounded border-2 my-3 p-3 shadow">
```

Comment:
```suggestion
            <form class="border rounded border-2 my-3 p-3 shadow">
```

Same thing here. No form, no form submit event.